geopy/geocoders/teleport.py

```diff
@@ -0,0 +1,283 @@
+"""
+:class:`Teleport` geocoder.
+"""
+
+from geopy.compat import urlencode
+from geopy.exc import GeocoderServiceError
+from geopy.geocoders.base import (
+    Geocoder,
+    DEFAULT_TIMEOUT,
+    DEFAULT_FORMAT_STRING,
+)
+from geopy.location import Location
+from geopy.util import logger
+try:
+    from urllib.parse import quote  # @UnresolvedImport @UnusedImport
+except:
+    from urllib import quote  # @Reimport
+
+
+__all__ = ("Teleport", )
+
+
+DEFAULT_FORWARD_EMBEDDINGS = (
+    "city:search-results/city:item/{city:country,city:admin1_division}"
+)
+DEFAULT_REVERSE_EMBEDDINGS = (
+    "location:nearest-cities/location:nearest-city" +
+    "/{city:country,city:admin1_division}"
+)
+
+
+try:
+    basestring
+except:
+    basestring = str  # @ReservedAssignment
+
+
+class Teleport(Geocoder):
+    """
+    Teleport geocoder and reverse geocoder
+
+    See API documentation at (https://developers.teleport.org/api/) for
+    details.
+
+    Example::
+
+        from geopy.geocoders import Teleport
+        from geopy.geocoders.teleport import DEFAULT_FORWARD_EMBEDDINGS
+
+        teleport = Teleport(forward_embeddings=DEFAULT_FORWARD_EMBEDDINGS +
```

Comment:
Could you please add this example to tests? I mean the `forward_embeddings` and `Teleport.get_embedded` parts, which are not covered in tests yet.